src/internal/m365/onedrive_test.go

```diff
@@ -45,118 +40,11 @@ var (
 	readPerm  = []string{"read"}
 )
 
-func mustGetDefaultDriveID(
-	t *testing.T,
-	ctx context.Context, //revive:disable-line:context-as-argument
-	ac api.Client,
-	service path.ServiceType,
-	resourceOwner string,
-) string {
-	var (
-		err error
-		d   models.Driveable
-	)
-
-	switch service {
-	case path.OneDriveService:
-		d, err = ac.Users().GetDefaultDrive(ctx, resourceOwner)
-	case path.SharePointService:
-		d, err = ac.Sites().GetDefaultDrive(ctx, resourceOwner)
-	default:
-		assert.FailNowf(t, "unknown service type %s", service.String())
-	}
-
-	if err != nil {
-		err = graph.Wrap(ctx, err, "retrieving drive")
-	}
-
-	require.NoError(t, err, clues.ToCore(err))
-
-	id := ptr.Val(d.GetId())
-	require.NotEmpty(t, id)
-
-	return id
-}
-
-type suiteInfo interface {
-	APIClient() api.Client
-	Tenant() string
-	// Returns (username, user ID) for the user. These values are used for
-	// permissions.
-	PrimaryUser() (string, string)
-	SecondaryUser() (string, string)
-	TertiaryUser() (string, string)
-	// ResourceOwner returns the resource owner to run the backup/restore
-	// with. This can be different from the values used for permissions and it can
-	// also be a site.
-	ResourceOwner() string
-	Service() path.ServiceType
-}
-
-type oneDriveSuite interface {
-	tester.Suite
-	suiteInfo
-}
-
-type suiteInfoImpl struct {
-	ac              api.Client
-	controller      *Controller
-	resourceOwner   string
-	secondaryUser   string
-	secondaryUserID string
-	service         path.ServiceType
-	tertiaryUser    string
-	tertiaryUserID  string
-	user            string
-	userID          string
-}
-
-func NewSuiteInfoImpl(
-	t *testing.T,
-	ctx context.Context, //revive:disable-line:context-as-argument
-	resourceOwner string,
-	service path.ServiceType,
-) suiteInfoImpl {
-	ctrl := newController(ctx, t, path.OneDriveService)
-
-	return suiteInfoImpl{
-		ac:            ctrl.AC,
-		controller:    ctrl,
-		resourceOwner: resourceOwner,
-		secondaryUser: tconfig.SecondaryM365UserID(t),
-		service:       service,
-		tertiaryUser:  tconfig.TertiaryM365UserID(t),
-		user:          tconfig.M365UserID(t),
-	}
-}
-
-func (si suiteInfoImpl) APIClient() api.Client {
-	return si.ac
-}
-
-func (si suiteInfoImpl) Tenant() string {
-	return si.controller.tenant
-}
-
-func (si suiteInfoImpl) PrimaryUser() (string, string) {
-	return si.user, si.userID
-}
-
-func (si suiteInfoImpl) SecondaryUser() (string, string) {
-	return si.secondaryUser, si.secondaryUserID
-}
-
-func (si suiteInfoImpl) TertiaryUser() (string, string) {
-	return si.tertiaryUser, si.tertiaryUserID
-}
-
-func (si suiteInfoImpl) ResourceOwner() string {
-	return si.resourceOwner
-}
-
-func (si suiteInfoImpl) Service() path.ServiceType {
-	return si.service
-}
+// type suiteInfoImpl struct {
```

Comment:
could you please remove this as well if it's not going to be revived?

Reply:
whoops, yup, didn't mean for those to hang around